File: core/src/backendServer/models/Bouncer.js
const constants = require('../settings/constants');
const Weapon = require('./Weapon');
const ProjectileType = require('./ProjectileType');

class Bouncer extends Weapon {
    constructor() {
        super(constants.BOUNCER_RELOAD_TIME, constants.BOUNCER_BULLETS_IN_CHAMBER,
            constants.BOUNCER_MAGAZINE_RELOAD_TIME, ProjectileType.BOUNCER)
    }
}

module.exports = Bouncer;

File: core/src/com/mygdx/game/model/weapon/Bouncer.kt
package com.mygdx.game.model.weapon

import com.mygdx.game.model.projectile.ProjectileType
import com.mygdx.game.settings.BOUNCER_BULLETS_IN_CHAMBER
import com.mygdx.game.settings.BOUNCER_RELOAD_TIME

class Bouncer : Weapon(BOUNCER_RELOAD_TIME, BOUNCER_BULLETS_IN_CHAMBER, ProjectileType.BOUNCER)